.eslintrc.js

```diff
@@ -176,9 +185,22 @@ module.exports = {
 				'@typescript-eslint/restrict-template-expressions': 'off',
 				'@typescript-eslint/unbound-method': 'off',
 
+				// override @typescript-eslint/stylistic-type-checked to ignore consistent-indexed-object-style. Both are used as required.
+				'@typescript-eslint/consistent-indexed-object-style': 'off',
```

Comment:
If we did need to disable this rule, we shouldn't have to disable it in both the `jsTsGlobs` override and the `tsGlobs` override. The `jsTsGlobs` override should have covered both.

But as noted previously, I think we should enable the rule.

Reply:
I moved this to `jsTsGlobs` I understood your comment as still wanting the override. if that's not the case, I will remove it.

Reply:
My comment was on `@typescript-eslint/consistent-indexed-object-style` which you turned `off` in both the `jsTsGlobs` and the `tsGlobs`. Since `tsGlobs` is a subset of `jsTsGlobs` you effectively turned the rule off twice. Since you were going to change this anyway, it was to provide an understanding that you were doing something redundant.

However, it seems like you've moved the `prefer-nullish-coalescing` rule to `jsTsGlobs`, but that wasn't the one I commented on. That rule requires type checking, which isn't enabled for `jsTsGlobs`. It has to be moved back.

Reply:
Oh ok gotcha. Thank you. I moved `prefer-nullish-coalescing` back to tsGlobs.